sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -352,3 +360,14 @@
 .form-check-input {
   position: inherit;
 }
+
+.need-help-image {
```

Comment:
This content page is a more generic page that could be completely changed by the Ministry.
I would avoid adding so specific styles dedicated to a so specific purpose here.
Maybe we can create just a relative class here and deal with the spacing directly in the form.io using some bootstrap classes already available.
Same for welcome-image style.